src/dnm/bcf2Qcall.cc

```diff
@@ -149,9 +149,9 @@ int bcf_2qcall(const bcf_hdr_t *hdr, bcf1_t *rec, Trio t, qcall_t *mom_snp,
       return err;
 
     // get I16 values from INFO field
-    std::array<int, 16> anno;
+    std::array<int, 16> anno = std::array<int,16>();
```

Comment:
`std::array<int, 16> anno = std::array<int,16>();` is the name as `std::array<int, 16> anno;`.